core/services/ardupilot_manager/ArduPilotManager.py

```diff
@@ -64,7 +64,10 @@ async def auto_restart_ardupilot(self) -> None:
                 self.ardupilot_subprocess is not None and self.ardupilot_subprocess.poll() is not None
             ) or len(self.running_ardupilot_processes()) == 0
             needs_restart = self.should_be_running and (
-                (self.current_platform in [Platform.SITL, Platform.Navigator] and process_not_running)
+                (
+                    self.current_platform in [Platform.SITL, Platform.Navigator, Platform.NavigatorR3]
```

Comment:
Navigator stands for R4, R5 or both ? It may be better to have a name for the version and not a generic name.

Reply:
That part was written when we still had only R3 and R4, and I forgot to consider R5. Will do that specifically, as it is on R3.